modules/tribes/client/components/CirclePage.component.js

```diff
@@ -0,0 +1,190 @@
+                  {circle.count === 0 && t('No members yet')}
+                  {circle.count === 1 && t('One memeber')}
+                  {circle.count > 1 && t(`${circle.count} members`)}
+                </div>
+                {circle.description && (
+                  <div className="lead tribe-meta">{t(circle.description)}</div>
+                )}
+                <br />
+                <br />
+                <JoinButton tribe={circle} user={user} onUpdated={() => {}} />
+                &nbsp;
+                <a
+                  className="btn btn-lg btn-default"
+                  href={`/search?tribe=${circle.slug}`}
+                  type="button"
+                >
+                  {t('Find members')}
+                </a>
+                <br />
+                <br />
+                <p className="lead tribe-intro">
+                  {t(
+                    'Trustroots is built on communities. Share this page within your community and invite them to join!',
+                  )}
+                </p>
+                {circle.label && (
+                  <>
+                    {/* Twitter share button */}
+                    <div
+                      className="tribe-share"
+                      tr-share-twitter
+                      data-text={t(
+                        `Join to meet, host and get hosted by Trustroots circle ${circle.label} — I'm in!`,
+                      )}
+                    ></div>
+                    {/* FB share button */}
+                    <div className="tribe-share" tr-share-fb></div>
+                  </>
+                )}
+              </div>
+            )}
+            {/* For non authenticated members */}
+            {!user && (
+              <div className="col-xs-12 col-sm-offset-1 col-sm-7 col-md-6 col-lg-5">
+                <p className="lead tribe-pre">{t('Trustroots circle')}</p>
+                <h2 className="font-brand-regular tribe-title">
+                  {t(circle.label)}
+                </h2>
+                <div className="lead tribe-meta">
+                  {circle.count === 0 && t('No members yet')}
+                  {circle.count === 1 && t('One memeber')}
+                  {circle.count > 1 && t(`${circle.count} members`)}
+                </div>
+                <br />
+                <br />
+                <p className="lead tribe-intro">
+                  {t(
+                    `Trustroots is a travellers' community for sharing, hosting and getting people together.`,
+                  )}
+                  <br />
+                  <br />
+                  {t(
+                    'Join to meet, host and get hosted by this and other communities.',
+                  )}
+                  <br />
+                  <br />
+                  <a
+                    className="btn btn-lg btn-primary btn-action tribe-join"
+                    ui-sref="signup({'tribe': tribeCtrl.tribe.slug})"
```

Comment:
Since `ui-sref` is Angular router, we need to use `href=""` instead. We can just do this:
```js
href={`/signup?tribe=${circle.slug}`}
```